### ISSUE TITLE
Beta test: getting 500 errors on everything after logging in

### ISSUE THREAD

**Issue by aldraco** [FreeCodeCamp/freecodecamp#971]
![screen shot 2015-06-18 at 2 35 32 pm](https://cloud.githubusercontent.com/assets/9719214/8239066/7b7a7648-15c7-11e5-8a0f-d400a0a33903.png)

I'm getting this error for everything after accessing the map, including bonfires, waypoints, news, guide, etc. 

The first bonfire I clicked actually gave a 502 bad gateway error, but I can't replicate it. Everything I've clicked on the map since then just gives 500.

Map works great the first time it loads, and correctly displays my completed/uncompleted items. Login with twitter works great. After that, it locks up.

Typing the URL back into my address bar and trying to access the site again from a fresh tab still locks up.

**Comment by BerkeleyTrue** [comment]
Thanks @aldraco 

This one is like the lock ness monster. Still trying to figure out exactly what is the cause.

Seems like a bad controller somewhere is not properly returning or calling back. Keep reporting on this issue if you find more

**Comment by BerkeleyTrue** [comment]
This may have been fixed with https://github.com/FreeCodeCamp/freecodecamp/commit/28f346e830bccdab7e0332f438f9724b8adf7c00

Thanks for your help! If you have some time we would love it if you could test out s'more!

**Comment by dlwebdev** [comment]
I know this is old and closed, but I am getting this error in my local environment. It ran fine for one instance as I was able to make some changes and submit a PR. When I tried looking at another issue the next morning I started getting this and have not been able to solve it. I even changed the timeout to 20000 as suggested in the "fix" referenced above. 

Has anyone else experienced this and if so what was the solution?
